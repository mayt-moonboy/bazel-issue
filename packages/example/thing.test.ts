import { Thing } from './thing'

describe('thing', () => {
  describe('doThing', () => {
    test('should return thing string', () => {
      const thing = new Thing()
      const result = thing.doThing()
      expect(result).toMatchSnapshot()
    })
  })
})