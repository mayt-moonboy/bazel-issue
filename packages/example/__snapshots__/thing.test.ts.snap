// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`thing doThing should return thing string 1`] = `
{
    "stuff": "thing string",
}
`;